jsettlers.logic/src/main/java/jsettlers/ai/highlevel/WhatToDoAi.java

```diff
@@ -359,11 +412,13 @@ private int numberOfAvailableToolsForBuildingType(EBuildingType buildingType) {
 			return Integer.MAX_VALUE;
 		}
 		return aiStatistics.getNumberOfMaterialTypeForPlayer(materialType, playerId)
-				- aiStatistics.getNumberOfNotFinishedBuildingTypesForPlayer(buildingType, playerId)
-				- aiStatistics.getNumberOfUnoccupiedBuildingTypeForPlayer(buildingType, playerId);
```

Comment:
Method AIStatistics.getNumberOfUnoccupiedBuildingTypeForPlayer() is never used any more. Is this ok? From the name I would think that information is required somewhere. If it is ok, please remove that method.